lib/pypechain/pypechain/run_pypechain.py

```diff
@@ -0,0 +1,116 @@
+"""Script to generate typed web3.py classes for solidity contracts."""
+from __future__ import annotations
+
+import json
+import os
+import sys
+from pathlib import Path
+
+from jinja2 import Environment, FileSystemLoader
+from pypechain.utilities import avoid_python_keywords, is_abi_function, solidity_to_python_type
+from web3 import Web3
+from web3.types import ABIFunction
+
+
+def load_abi_from_file(file_path: Path):
+    """Loads a contract ABI from a file."""
+    with open(file_path, "r", encoding="utf-8") as file:
+        return json.load(file)["abi"]
+
+
+def main(abi_file_path: str, output_file_path: str):
+    """Generates class files for a given abi."""
+    ### Set up the Jinja2 environment
+    # Determine the absolute path to the directory containing your script.
+    script_dir = os.path.dirname(os.path.abspath(__file__))
+    # Construct the path to your templates directory.
+    templates_dir = os.path.join(script_dir, "templates")
+    env = Environment(loader=FileSystemLoader(templates_dir))
+    template = env.get_template("contract.jinja2")
+
+    _web3 = Web3()
```

Comment:
we typically follow Web3's convention to call this var `web3`. I assume you added the underscore bc of an earlier lint namespace error (this has come up a lot in the past), but it looks like you can remove it now.